src/Microsoft.OData.Core/UrlValidation/ODataUrlValidationMessage.cs

```diff
@@ -0,0 +1,88 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataUrlValidationMessage.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.OData.UriParser.Validation
+{
+    /// <summary>
+    /// Severity of an OData Url Validation Message
+    /// </summary>
+    public enum Severity

```

Comment:
David adds the same in the "Edm" side.
So, maybe we should reuse that.

Reply:
Although possible, I thought it was a bit weird to have to reference the Edm namespace in order to parse the Url.  I did align both to use the same members.

---
In reply to: [434872325](https://github.com/OData/odata.net/pull/1769#discussion_r434872325) [](ancestors = 434872325)